src/components/modules/toolbar/index.ts

```diff
@@ -466,12 +467,27 @@ export default class Toolbar extends Module<ToolbarNodes> {
    * Enable bindings
    */
   private enableModuleBindings(): void {
+    this.readOnlyMutableListeners.on(this.nodes.settingsToggler, 'dragstart', (event: DragEvent) => {
+      this.Editor.BlockManager.currentBlock = this.hoveredBlock;
+      this.Editor.BlockManager.currentDraggingBlock = this.hoveredBlock;
+
+      this.nodes.settingsToggler.classList.add(this.CSS.settingsTogglerDragging);
+
+      this.tooltip.hide(true);
+    }, true);
+
+    this.readOnlyMutableListeners.on(this.nodes.settingsToggler, 'dragend', (event: DragEvent) => {
+      this.Editor.BlockManager.currentDraggingBlockIndex = -1;
```

Comment:
You can create a method at the BlockManager for that. Just like a `dropPointer()`. Or maybe it would better to add the `currentDraggingBlockIndex = -1` to the `dropPointer`? Should we clear the `currentBlock` index here?

Reply:
Mmmm I believe they should be kept as separate pointers, since if we implement a way to drag & drop several selected blocks.. you shouldn't drop the selection after dragging. Am I right ?